client/utils/useHistoricalLockupToasts.js

```diff
@@ -0,0 +1,141 @@
+import { useEffect, useState } from "react";
+import { sample, random, maxBy } from "lodash";
+import { useStore } from "utils/store";
+import { toast } from "react-toastify";
+import { utils } from "ethers";
+import { truncateEthAddress } from "utils";
+
+const useHistoricalLockupToasts = () => {
+  const { web3Provider, contracts } = useStore();
+  // 13.13 seconds is average block time (on 7.7.2022) ~
+  const blocksToLookBack = 1300; // roughly 2 days
+  const SECONDS_IN_A_MONTH = 2629743;
```

Comment:
We need to check that we use the same value across the project. See Discord comment: https://discord.com/channels/404673842007506945/758123420230615080/994609704477134878

Reply:
good point changing to: 2592000